Add Entries for HTML Text Formatting and <b>,<i>, and <u> Tags (#602)
Codecademy/docs: content/html/concepts/tags/terms/b/b.md

---
Title: '<b>'

> Description: 'Used to draw the reader''s attention to a section of HTML text, typically rendered in boldface.'

Reviewer: ```suggestion
Description: 'Used to draw attention to a section of text, usually rendered in boldface.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:

>   - 'Tags'

Reviewer: Wondering if we could add at least one more meta-`Tag`? What about `Style`?

CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `<b>` element is used to draw the reader's attention to the marked text without assigning any additional importance to it. This is typically done by making the text bold.

**Note:** While historically this tag was used just to set text as bold, it should now be avoided for solely styling text. Modern best practice to use [CSS](https://www.codecademy.com/resources/docs/css) for text formatting. The CSS element used for making boldface text is [`font-weight`](https://www.codecademy.com/resources/docs/css/typography/font-weight).

## Syntax

```html

> <u><!-- text which should be annotated goes here --></u>

Reviewer: ```suggestion
<b><!-- text which should be annotated goes here --></b>
```

```

> The `<b>` tag should only contain text content or other tags meant for text content.

Reviewer: ```suggestion
The `<b>` tag should only contain text or other tags meant for text content.
```

---
Title: '<b>'
Description: 'Used to draw attention to a section of text, usually rendered in boldface.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Tags'
  - 'Style'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<b>` element is used to draw the reader's attention to the marked text without assigning any additional importance to it. This is typically done by making the text bold.

Author: The suggestion isn't quite accurate anymore. The current specs specifically don't want the b tag to just be used to make something bold.
